testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md

```diff
@@ -0,0 +1,21 @@
+|------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Launch Create a Gem experience           | 1.  Enter the Gem Catalog<br>2.  Select the hamburger menu in the top right corner<br>3.  Select "Create a New Gem"                                                                                            | Dialog changes to Create a New Gem step 1 "Gem Setup"                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| Step 1 - Gem Setup Options               | 1.  Launch the Create a Gem experience<br>2.  Observe the options in Step 1                                                                                                                                    | Dialog displays options for:<br><br>*   Default Gem<br>*   Prebuilt Gem<br>*   Asset Gem<br>*   C++ Tool Gem<br>*   Python Tool Gem<br>*   Choose existing template                                                                                                                                                                                                                                                                                                                         |
+| Step 2 - Gem Details                     | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next                                                                                                                | *   All fields can be edited<br>    *   Gem Display Name<br>    *   Gem System Name<br>    *   Gem Summary<br>    *   Requirements<br>    *   License<br>    *   License URL<br>    *   Origin<br>    *   Origin URL<br>    *   Global Gem Tags<br>    *   Optional Global Gem Tag<br>    *   Optional Global Gem Tag<br>    *   User-defined Gem Tags<br>    *   Gem Location<br>    *   Gem Icon Path<br>    *   Documentation URL<br>*   Global Gem Tags field populates with gem fields |
```

Comment:
Order between gem display name and gem system name need to be swapped.

We currently do not have Origin and Origin URL fields.

In the gem.json file, there is no distinction between User and Global gem tags. We will be just creating a single tag input field, and global gems will be implemented as an auto-completion mechanism. See this PR: https://github.com/o3de/o3de/pull/12501